src/uploadBuilderBuilder.js

```diff
@@ -0,0 +1,44 @@
+export const uploadBuilderBuilder = versionSuffix => {
+  const command = `upload${versionSuffix}`;
+  const pathOption = `<path${versionSuffix === '-proguard' ? '' : 's..'}>`;
+  return (args) => {
+    args
+      .usage(`\nUsage: logrocket ${command} -r <release> ${pathOption}`)
```

Comment:
That's already included in the usage string for the base CLI, which I think means it should be included for each of the commands.

Reply:
I think that would be neat :)

Reply:
So, as previously mentioned, it looks like the apikey being in the usage string for the base of the CLI only includes it in the help for the output when no command is included

```
% ../logrocket-cli/bin/logrocket --help              

Usage: logrocket [-k <apikey>] <command> [<args>]

Commands:
  release <version>        Create a new release
  upload <paths..>         Upload JavaScript sourcemaps for a release
  upload-mobile <paths..>  Upload debug file(s) for a mobile release

Options:
  -k, --apikey  Your LogRocket API key                       [string] [required]
  -h, --help    Show help                                              [boolean]

mckenzie@MacBook-Pro-4 logrocket % 
```

BUT it does still include the apikey in the options for the individual commands and specify that it's required.
```
 % ../logrocket-cli/bin/logrocket upload-mobile --help
Usage: logrocket upload-mobile -r <release> -p <platform> <paths..>

Options:
  -k, --apikey       Your LogRocket API key                  [string] [required]
  -r, --release      The release version for these files     [string] [required]
  -p, --platform     The mobile platform of this file (or files)
                                          [required] [choices: "ios", "android"]
  --max-retries      Failed upload retry limit (0 disables)[number] [default: 0]
  --max-retry-delay  Maximum delay between retries in ms
                                                       [number] [default: 30000]
  -h, --help         Show help                                         [boolean]
```
Do we feel okay about this, or does it feel important to include the apikey in the usage string for each of the individual commands as well?

Reply:
do you happen to what the significance of square brackets is? I'm wondering whether it would be accurate to include them around `<paths..>`.

Reply:
Not actually sure what the convention is around those - iirc from the yargs docs that `[value]` means optional and `<value>` means required - but since the square brackets are around the angle ones, my guess is it indicates the positioning isn't required; in which case you're right that we shouldn't add them for `paths` 👍